experimental/generation/generator/packages/library/templates/standard/custom.lu.lg

```diff
@@ -0,0 +1,12 @@
+[generator.lg](../generator.lg)
+
+# filename
+- ${understandingDir()}${locale}/form/${prefix}-custom.${locale}.lu
+
+# template
+-```
+> Add custom utterances
+> Label with: Add(), Remove(), Help(), Clear(), Show() and Help()
```

Comment:
Maybe add all of the labels and the hierarchy, i.e. Values: BreadValue, CheeseValue
Properties: BreadProperty(BreadValue), Length(dimension, number)
Operations: Add(PROPERTY), ...

Reply:
We still need more documentation here.

---
In reply to: [587060618](https://github.com/microsoft/BotBuilder-Samples/pull/3104#discussion_r587060618) [](ancestors = 587060618)